### PR TITLE
fix(web): multiple fixes for the webUI

### PR DESCRIPTION
**Pull request by martabal**
fix for:
- #8351
- #8344

### CODE REVIEW

File: web/src/lib/components/asset-viewer/asset-viewer.svelte [hunk 1]
```diff
@@ -588,7 +591,7 @@
             <VideoViewer
               assetId={previewStackedAsset.id}
               on:close={closeViewer}
-              on:onVideoEnded={handleVideoEnded}
+              on:onVideoEnded={() => navigateAsset()}
```
Comment (alextran1502) [inline web/src/lib/components/asset-viewer/asset-viewer.svelte:594]:
```suggestion
              on:onVideoEnded={navigateAsset}
```

File: web/src/lib/components/asset-viewer/asset-viewer.svelte [hunk 2]
```diff
@@ -621,7 +624,7 @@
             <VideoViewer
               assetId={asset.id}
               on:close={closeViewer}
-              on:onVideoEnded={handleVideoEnded}
+              on:onVideoEnded={() => navigateAsset()}
```
Comment (alextran1502) [inline web/src/lib/components/asset-viewer/asset-viewer.svelte:627]:
```suggestion
              on:onVideoEnded={navigateAsset}
```

File: web/src/lib/components/asset-viewer/asset-viewer.svelte (hunk 1 above)
Comment (martabal) [reply web/src/lib/components/asset-viewer/asset-viewer.svelte:594]: No you can't because `e: CustomEvent<void>` != `order?: 'previous' | 'next'`

**Review by jrasm91 — APPROVED** [review]
lgtm

**Review by danieldietzler — APPROVED** [review]

**Pull request merged**